theano/gpuarray/subtensor.py

```diff
@@ -20,6 +20,22 @@
                         infer_context_name)
 
 
+iadd_reg = {}
+
+
+def get_iadd(a, b):
+    key = (a.type.dtype, b.type.dtype, a.type.context)
+    if key not in iadd_reg:
+        if a.dtype == 'float16' or b.dtype == 'float16':
+            raise NotImplementedError('float16 is not supported by pygpu '
+                                      'elemwise')
```

Comment:
Is this comment still true? https://github.com/Theano/Theano/issues/1471 tell that Elemwise and Subtensor are supported in float16 in the new back-end.